src/Agent.Worker/Extensions/VariablesExtension.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.TeamFoundation.DistributedTask.WebApi;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Extensions
+{
+    public static class VariablesExtension
+    {
+        public static Dictionary<string, VariableValue> ToWebApiVariables(this IEnumerable<Variable> outputVariables)
+        {
+            var webApiVariables = new Dictionary<string, VariableValue>(StringComparer.OrdinalIgnoreCase);
+
+            if (outputVariables == null || !outputVariables.Any())
+            {
+                return webApiVariables;
+            }
+
+            foreach (var outputVariable in outputVariables)
```

Comment:
please change the var to actual type. the type is not clear from right hand side